src/mpc.jl

```diff
@@ -0,0 +1,10 @@
+module MPC
+
+mutable struct Optimizer <: MOI.AbstractOptimizer
+end
+
+function MOI.optimize!(optimizer::Optimizer)
```

Comment:
I guess you didn't mean to push this

Reply:
No idea where this came from... I'll fix it